kubernetes/ansible-kube/environments/kube/files/invoker.yml

```diff
@@ -0,0 +1,76 @@
+---
+apiVersion: apps/v1beta1
+kind: StatefulSet
+metadata:
+  name: invoker
+  namespace: openwhisk
+  labels:
+    name: invoker
+spec:
+  replicas: 1
+  serviceName: "invoker"
+  template:
+    metadata:
+      labels:
+        name: invoker
+    spec:
+      restartPolicy: Always
+
+      volumes:
+      - name: cgroup
+        hostPath:
+          path: "/sys/fs/cgroup"
+      - name: runc
+        hostPath:
+          path: "/run/runc"
+      - name: dockerrootdir
+        hostPath:
+          path: "/var/lib/docker/containers"
+      - name: dockersock
+        hostPath:
+          path: "/var/run/docker.sock"
+      - name: apparmor
+        hostPath:
+          path: "/usr/lib/x86_64-linux-gnu/libapparmor.so.1"
+
+      containers:
+      - name: invoker
+        imagePullPolicy: IfNotPresent
+        image: danlavine/whisk_kube_invoker
```

Comment:
Now that you mention it, I actually might be able to do something similar. I was having some trouble setting the command properly for Kube, but I solved it when setting the command to download all of the images required for the invoker. Let me go back and try to inline everything on this [Kube file](https://github.com/DanLavine/openwhisk-devtools/blob/kube/kubernetes/ansible-kube/environments/kube/files/invoker.yml).